Makefile

```
@@ -41,7 +41,7 @@ info:
 	@echo " $(foreach PLUGIN_TYPE,$(PLUGIN_TYPES), $(PLUGIN_TYPE):\n $(foreach PLUGIN,$(basename $(notdir $(_$(PLUGIN_TYPE)))),   - $(PLUGIN)\n)\n)"
 
 lint: $(MANIFEST) | tests/test_playbooks/vars/server.yml
-	yamllint -f parsable tests/test_playbooks
+	yamllint -f parsable tests/test_playbooks roles
```

```suggestion
	yamllint -f parsable tests/test_playbooks roles
	ansible-lint -v roles/*
```

this might require adding `ansible-lint` to `requirements-lint.txt`